Website/Packages/datax-pipeline/src/modules/flowDefinition/components/rule/rulesSettingsContent.jsx

```diff
@@ -319,8 +319,7 @@ const rightPaneStyle = {
 const listContainerStyle = {
     backgroundColor: Colors.white,
     border: `1px solid ${Colors.neutralTertiaryAlt}`,
-    flex: 1,
-    overflowY: 'auto'
+    flex: 1
```

Comment:
is this change intentional? this UI is related to rules and not batching/scheduling, did you reuse this PR to fix some other bug? By remove the overflowY: 'auto' statement, what does this fix?

Reply:
no, it wasn't intentional. reverted.